tinyagentos/projects/canvas/store.py

```
@@ -0,0 +1,201 @@
+            if col in patch:
+                sets.append(f"{col} = ?")
+                params.append(patch[col])
+        if "payload" in patch:
+            sets.append("payload = ?")
+            params.append(json.dumps(patch["payload"]))
+        if not sets:
+            existing = await self.get_element(element_id)
+            if existing is None:
+                raise ValueError(f"element not found: {element_id}")
+            return existing
+        sets.append("updated_at = ?"); params.append(time.time())
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)